.github/workflows/test_osx_python.yml

```diff
@@ -15,17 +15,21 @@ jobs:
     name: "unit_test (${{ matrix.runs_on }}, ${{ matrix.python }})"
     runs-on: ${{ matrix.runs_on }}
     strategy:
+      fail-fast: false
       matrix:
         # macos-13 runs on Intel
-        # macos-14 runs on Apple Silicon
-        runs_on: ['macos-13', 'macos-14']
-        python: ['3.13']
+        # macos-latest and macos-26 run on Apple Silicon
+        runs_on: ['macos-13', 'macos-26', 'macos-latest']
+        python: ['3.x']
     steps:
-    - uses: actions/checkout@v4
+    - if: matrix.runs_on == 'macos-26'
+      run: xcodebuild -downloadComponent MetalToolchain
```

Comment:
@misl6 Lines 25 and 26 are required make things work on macOS 26 on GitHub Actions.